templates/pages/category.html

```diff
@@ -25,9 +25,15 @@ <h1 class="page-heading">{{category.name}}</h1>
 {{{snippet 'categories'}}}
 <div class="page">
     {{#or category.subcategories category.faceted_search_enabled category.shop_by_price}}
-        <aside class="page-sidebar" id="faceted-search-container">
+        {{#if category.subcategories}}
             {{> components/category/sidebar}}
-        </aside>
+        {{else if category.faceted_search_enabled}}
+            {{> components/category/sidebar}}
+        {{else if category.shop_by_price}}
+            {{#if theme_settings.shop_by_price_visibility}}
+                {{> components/category/sidebar}}
+            {{/if}}
+        {{/if}}
```

Comment:
@junedkazi fixed to handle the cases separately. if you have suggestions or if you can point me in the direction of whatever templating lang this is so i can read up.

Reply:
This is https://handlebarsjs.com